src/main/java/org/thoughtcrime/securesms/LogViewFragment.java

```diff
@@ -256,6 +258,8 @@ private static String buildDescription(LogViewFragment fragment) {
       }
 
       final String token = FcmReceiveService.getToken();
+      boolean notifPermGranted = PermissionChecker.checkSelfPermission(context, Manifest.permission.POST_NOTIFICATIONS) == PermissionChecker.PERMISSION_GRANTED;
+      builder.append("notifications-permission=").append(notifPermGranted).append("\n");
```

Comment:
i would be more explicit and on point here, using the same wording. things are complex enough :)

```suggestion
      builder.append("post-notifications-granted=").append(notifPermGranted).append("\n");
```